src/gdm/temporal_models.py

```diff
@@ -0,0 +1,94 @@
+from datetime import date
+from typing import Any
+
+from pydantic import field_validator
+from gdm.distribution.distribution_system import DistributionSystem
+from rich.console import Console
+from infrasys import Component
+from rich.table import Table
+from loguru import logger
+from uuid import UUID
+
+
+class PropertyEdit(Component):
+    name: str
+    value: Any
+    component_uuid: UUID
+
+
+class ModelUpdate(Component):
+    name: str = ""
+    update_date: date
+    additions: list[UUID] = []
+    edits: list[PropertyEdit] = []
+    deletions: list[UUID] = []
+
+
+class ModelUpdates(Component):
```

Comment:
Looks like you are using the ModelUpdates class just for the sake of sorting. I think you can do the sort in the function where you are using it.